lib/prometheus-metrics.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+class PrometheusMetrics {
+  constructor (circuit) {
+    this.circuit = circuit;
+    // this.client = require('prom-client');
+    this.collect();
+  }
+
+  collect (prefix) {
+    this.collectDefaultMetrics = this.client.collectDefaultMetrics({ prefix: prefix, timeout: 5000 });
+    this.circuit.on('open', () => {
+      this.clear();
+    });
+    this.circuit.on('halfOpen', () => {
+      this.clear();
+    });
+  }
```

Comment:
We should probably add a snapshot listener that resets all of the metrics for each snapshot. We don't want counters to increase indefinitely. Instead, reset the counter for each snapshot bucket.

```suggestion
  this.circuit.status.on('snapshot', _ => this.counters.forEach( counter => counter.reset() );
```